app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -1911,6 +1911,309 @@ class DatabaseMigrationAndroidTest {
       assertThat(it.string(newColumnName)).isEqualTo("2017-09-25T11:20:42.008Z")
     }
   }
+
+  @Test
+  fun migrate_prescription_from_35_to_36() {
+    val db_v35 = helper.createDatabase(version = 35)
+
+    val tableName = "PrescribedDrug"
+    val uuid = "drug-uuid"
+    db_v35.execSQL("""
+      INSERT INTO $tableName VALUES(
+      '$uuid',
+      'drug',
+      'dosage',
+      'rxNormCode',
+      0,
+      1,
+      'patientUuid',
+      'facilityUuid',
+      'PENDING',
+      'created-at',
+      'updatedAt',
+      'null',
+      'recorded-at')
+    """)
+
+    val db_v36 = helper.migrateTo(version = 36)
+
+    db_v36.query("SELECT * FROM $tableName").use {
+      it.moveToNext()
+      assertThat(it.columnCount).isEqualTo(12)
+      assertThat(it.count).isEqualTo(1)
+      assertThat(it.string("uuid")).isEqualTo(uuid)
+    }
+  }
+
+  @Test
+  fun migrate_Appointment_from_35_to_36() {
```

Comment:
The method name here is inconsistent with the rest of the test methods since this is the only one where the table name is in caps. Change this to match the others?